ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -173,7 +173,7 @@ class LargeTitleView: UIView {
         let preferredFallbackImageStyle: MSFAvatarStyle = style == .light ? .default : .accent
         let avatar = MSFAvatar(style: preferredFallbackImageStyle,
                                size: Constants.avatarSize)
-        let avatarState = avatar.state
+        let avatarState: MSFAvatarState = avatar.state
```

Comment:
No clue why this is necessary. Seems like a compiler bug, since `avatar.state` is explicitly declared as `MSFAvatarState!`, unless I'm missing something subtle.

Note that it's fine if I remove the `@objc` part, which is only more confusing.